vNext/src/OrleansCounterControl/OrleansCounterControl.csproj

```diff
@@ -0,0 +1,42 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+  <PropertyGroup Label="NuGet">
+    <PackageId>Microsoft.Orleans.CounterControl</PackageId>
+    <Title>Microsoft Orleans Performance Counter Tool</Title>
+    <Description>Tool that registers performance counter categories for performance counters published by Orleans runtime.</Description>
+  </PropertyGroup>
+
+  <PropertyGroup>
+    <TargetFramework>net461</TargetFramework>
```

Comment:
This works with the new tooling already? Because with tooling 1.X it didn't allow me to reference .net standard 1.5 from .net 4.6.1
But I understand the plan is that this would be supported.